### PR TITLE
docs(ai): Creates AI guidelines.

### LINKED ISSUE #4142: PF AI guidelines - UX AI Design Guidelines and Principles
**Needs more content** cc Abi

Need to create a new section on PF to house UXD AI ethics and guidelines.

### PR DESCRIPTION
**Pull request by edonehoo**
Closes #4142

### CODE REVIEW

**Comment by edonehoo** [comment]
hi! @abigaeljamie @kevinhatchoua @andrew-ronaldson I've got a draft up for adding initial AI guidelines to the site. I worked off of the doc that Abi and Kevin put together, but made some adjustments to cut down the length a little and group things around in different ways. Very open to any feedback/ideas to adjust based on what you all were envisioning!

File: packages/documentation-site/patternfly-docs/content/AI/about-ai.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+To guide the use of AI in design practices, we have developed AI-driven design principles and guidelines that consider how:
+
+- The way that we design is just as important as what we design.
+- AI experiences can impact people in both desired and undesired ways.
+
+## AI principles 
+
+These design principles build an ethics-first framework and guide the use of AI with PatternFly. AI systems should be: 
+- Accountable. 
+- Transparent.
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/AI/about-ai.md:17]:
```suggestion
- Explainable.
- Transparent. 
```
Add

**Comment by andrew-ronaldson** [comment]
@bekah-stephens has a design issue open to look at a layout for this content. https://github.com/patternfly/patternfly-design/issues/1322

File: packages/documentation-site/patternfly-docs/content/AI/about-ai.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- The way that we design is just as important as what we design.
+- AI experiences can impact people in both desired and undesired ways.
+
+## AI principles 
+
+These design principles build an ethics-first framework and guide the use of AI with PatternFly. AI systems should be: 
+- Accountable. 
+- Explainable.
+- Transparent. 
+- Fair
```
Comment (abigaeljamie) [inline packages/documentation-site/patternfly-docs/content/AI/about-ai.md:19]:
```suggestion
- Fair. 
```

**Review by abigaeljamie — APPROVED** [review]
These look great! I added a missing period, but otherwise no feedback :D

**Comment by kmcfaul** [comment]
Wait to merge this until chatbot is ready

File: packages/documentation-site/patternfly-docs/patternfly-docs.source.js [hunk 3]
```diff
@@ -31,6 +31,8 @@ module.exports = (sourceMD, sourceProps, sourceFunctionDocs) => {
 
     sourceMD(path.join(contentBase, 'tokens/**/*.md'), 'tokens');
 
+    sourceMD(path.join(contentBase, 'AI/**/*.md'), 'AI');
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/patternfly-docs.source.js:34]:
```suggestion
    sourceMD(path.join(contentBase, 'PatternFly-AI/**/*.md'), 'PatternFly-AI');
```

**Pull request merged**